apps/api/src/order/order.controller.ts

```diff
@@ -17,6 +29,26 @@ export class OrderController {
     private jwtService: JwtService,
   ) {}
 
+  @UseGuards(AuthGuard)
+  @Roles(Role.CONSUMER)
+  @UseGuards(RolesGuard)
+  @CheckAbilities({ action: Action.Read, subject: Order })
+  @UseGuards(AbilityGuard)
+  @Get('/')
+  async findOne(@Request() req): Promise<Order> {
+    return this.orderService.findOne(req.id);
+  }
+
+  @UseGuards(AuthGuard)
+  @Roles(Role.CONSUMER)
+  @UseGuards(RolesGuard)
+  @CheckAbilities({ action: Action.Read, subject: Order })
+  @UseGuards(AbilityGuard)
+  @Get('get-all-orders')
```

Comment:
same with this one, url will be _order/get-all-orders_ (redundant)

can be simplified to _order/get-all_